cmd/istioctl/main.go

```diff
@@ -99,6 +99,7 @@ var (
 			if len(args) != 0 {
 				return fmt.Errorf("create takes no arguments")
 			}
+			c.SilenceUsage = true
```

Comment:
I have seen a Cobra user saying the best way to do this is to set `SilenceUsage: true` when the root Cmd is defined, and explicitly print the usage messages myself.  I didn't want to introduce a lot of changes to code that I did not write.  Setting this explicitly after validation happens seems clean enough.